shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptGeneratorBaseTest.java

```diff
@@ -76,6 +89,40 @@ protected static InsertStatement createInsertStatement() {
         return result;
     }
     
+    /**
+     * SQL: UPDATE t_user SET pwd = '654321' WHERE name = 'LiLei' AND pwd = '123456'.
+     */
+    protected static UpdateStatementContext createUpdatesStatementContext() {
+        MySQLUpdateStatement mySQLUpdateStatement = new MySQLUpdateStatement();
+        mySQLUpdateStatement.setTableSegment(createTableSegment(TABLE_NAME));
+        mySQLUpdateStatement.setWhere(createWhereSegment());
+        mySQLUpdateStatement.setSetAssignment(createSetAssignmentSegment());
+        return new UpdateStatementContext(mySQLUpdateStatement);
+    }
+    
+    /**
+     * SQL-template: WHERE name = 'LiLei' AND pwd = '123456'.
```

Comment:
Please remove useless javadoc in unit test.

Reply:
ok